test/plugin.test.js

```diff
@@ -108,6 +108,58 @@ QUnit.test('no progress for 45 seconds is an error', function(assert) {
   assert.strictEqual(this.player.error().type, 'PLAYER_ERR_TIMEOUT');
 });
 
+QUnit.test('the plugin cleans up after its previous incarnation when called again',
```

Comment:
This is pretty low-level. Is there a way to test this that stays out of the internals a bit more? For instance, could you listen for timeout errors and count whether one or two of them are fired after the timeout expires?